site/js/worklets/theme.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * Site Theme setter and getter
+ */
+export class Theme {
+  /**
+   *
+   * @param {DOMElement} elem
+   * @param {string} name
+   * @return {void}
+   */
+  constructor(elem, name) {
+    this._elem = elem;
```

Comment:
Because of the way `name` works, I don't see this being usable by other elements. We should just use `document.body` directly here

Reply:
I see it. updated.